lib/features/interactive.js

```diff
@@ -0,0 +1,112 @@
+var debug = require('debug')('docker-worker:features:interactive');
```

Comment:
this is a nit, but since we're adopting es6/7 goodies elsewhere, I'm not sure why we're not using import and let